src/modules/m_cgiirc.cpp

```diff
@@ -38,21 +38,7 @@ enum
 static void ChangeIP(LocalUser* user, const irc::sockets::sockaddrs& sa)
```

Comment:
This is pretty much empty now so it can be removed and the two lines of code inlined to the places that call it.